azure-pipelines.yml

```diff
@@ -82,23 +82,23 @@ jobs:
     displayName: Build wheel and install pyvista
 
   - script: |
-      .ci/setup_headless_display.sh
+      sudo apt-get update && sudo apt-get install libgl1-mesa-glx xvfb -y
```

Comment:
Could you explain why `setup_headless_display.sh` is removed?

Reply:
We're using xvfb-run now.  Reason is there's unclosed plots from doctest-modules with VTK 9 that remain open and cause doctest-modules-local-namespace to set fault. This hasn't been an issue until I added a few more examples in this PR, which bring it over the limit of 255 simultaneous windows.  By using xvfb-run, we open up one display for each doctest, allowing us to actually run the doctests.